libp2p/switch.nim

```diff
@@ -260,21 +272,21 @@ proc newSwitch*(peerInfo: PeerInfo,
   switch.mount(identity)
   return switch
 
-proc isConnected*(s: Switch, peerInfo: PeerInfo): bool
-  {.deprecated: "Use PeerID version".} =
-  not isNil(peerInfo) and isConnected(s, peerInfo.peerId)
+# proc isConnected*(s: Switch, peerInfo: PeerInfo): bool
```

Comment:
remove? uncomment?

Reply:
Probably remove, it's been deprecated for a while and we don't use it anywhere anymore, but waku might for example...